src/auth.tsx

```diff
@@ -59,9 +60,153 @@ export interface ClaimsCheckProps {
   user: firebase.User;
   fallback: React.ReactNode;
   children: React.ReactNode;
-  requiredClaims?: { [key: string]: any };
+  requiredClaims: { [key: string]: any };
 }
 
+interface ClaimCheckErrors {
+  [key: string]: ReactFireError[];
```

Comment:
Do we want to restrict developers to ReactFireError for custom validators? Or should we do any[]?

Reply:
good point, made this an `any[]`